src/components/ListingCard/ListingCard.js

```diff
@@ -98,8 +110,10 @@ export const ListingCardComponent = props => {
               longWordClass: css.longWord,
             })}
           </div>
-          <div className={css.authorInfo}>
-            <FormattedMessage id="ListingCard.hostedBy" values={{ authorName }} />
+          <div className={css.certificateInfo}>
+            {certificate === '200h' || certificate === '500h' ? (
```

Comment:
Avoid spreading config info outside of marketplace-custom-config.js.
Either check against single value "none" or completely different approach would be to add only certificates to `certificate` config. So, the 'none' case would then be the situation where `publicData.certificate` is undefined.